pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -284,6 +338,15 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 		return true, err
 	}
 
+	// determine the amount of time to delay before creating the next installer pod.  We delay to avoid an LB outage (see godoc on minReadySeconds)
+	sleepTime := c.timeToWaitBeforeInstallingNextPod(ctx, operatorStatus.NodeStatuses)
+	if sleepTime > 0 {
+		select {
+		case <-ctx.Done():
+		case <-time.After(sleepTime):
+		}
+	}
+
 	for l := 0; l < len(operatorStatus.NodeStatuses); l++ {
```

Comment:
the code below ensures that only one node can transition to a new revision at a time, right?

Reply:
> the code below ensures that only one node can transition to a new revision at a time, right?

I can find a ref if you want it, but yes, we check the previous installer pod status

Reply:
just wanted to make sure that `durationPodHasBeenReady` counting makes sense. thanks.